pkg/zson/docs/spec.md

```diff
@@ -204,6 +205,10 @@ by a value encoding.  Here is a pseudo-grammar for value encodings:
 
 [1] - [JavaScript character escaping rules](https://tc39.es/ecma262/#prod-EscapeSequence)
 
+A channel is a 16-bit integer used to indicate sub-streams of values within the zson stream.
+This is useful, for example, when analytics performs two or computations on the 
```

Comment:
I assume that was meant to say "two or more computations"?

Reply:
yes!

Reply:
fixed